sound/hda/intel-dsp-config.c

```diff
@@ -165,6 +169,10 @@ static const struct config_entry config_table[] = {
 			{}
 		}
 	},
+	{
+		.flags = FLAG_SOF | FLAG_SOF_ONLY_IF_SOUNDWIRE,
+		.device = 0x9dc8,
+	},
```

Comment:
Can we merge the flags to FLAG_SOF | FLAG_SOF_ONLY_IF_DMIC | FLAG_SOF_ONLY_IF_SOUNDWIRE in such cases?
So we will use SOF no matter DMIC or SOUNDWIRE is selected.

Reply:
yes, good suggestion @bardliao, will update.